### PR TITLE
Remove the requirement for 3 touch points

### PR DESCRIPTION
**Pull request by mpowelson**
I'm not sure what the ultimate plan is for this (or any) feature. However, until the master plan for what this thing is supposed to look like is released, I'm proposing we remove the requirement to have 3 touch points.

### CODE REVIEW

**Comment by mpowelson** [comment]
@DavidMerzJr @marip8 ping

File: opp_gui/src/widgets/tool_path_planner_widget.cpp [hunk 1]
```diff
@@ -303,7 +305,7 @@ void ToolPathPlannerWidget::saveModel()
   using TouchPointMap = std::map<std::string, opp_msgs::TouchPoint>;
   TouchPointMap touch_points = touch_point_editor_->getPoints();
   TouchPointMap verification_points = verification_point_editor_->getPoints();
-  if (touch_points.size() < 3 || verification_points.size() < 3)
+  if (touch_points.size() < MIN_TOUCH_POINTS || verification_points.size() < MIN_VERIFICATION_POINTS)
   {
     QMessageBox::warning(
         this, "Invalid Model Definition", "Ensure at least 3 touch points and 3 verification points have been defined");
```
Comment (marip8) [inline opp_gui/src/widgets/tool_path_planner_widget.cpp]: I would also change this message to indicate the number of touch and verification points that are required

**Review by marip8 — CHANGES_REQUESTED** [review]

**Comment by marip8** [comment]
I created an issue (#18) to discuss the fate of these features, but I think this is a good stop-gap measure for users who don't want/need to create the points

**Review by marip8 — APPROVED** [review]

**Comment by drchrislewis** [comment]
@marip8 Who has authority to merge? I don't seem to. This one is long overdue for this simple fix.

**Comment by DavidMerzJr** [comment]
@drchrislewis

@marip8 and @jrgnicho are (to my knowledge) the only ones who have merge rights on this repo.

**Comment by marip8** [comment]
It appears CI is failing because of a clang formatting issue. I'll update and merge today

**Comment by drchrislewis** [comment]
@marip8 If you have permissions on this repo, please accept this PR. Also please give me permissions. I have several PR's between myself and Collin that we'd like to review and approve.

**Pull request merged**